packages/extension-chakra-storefront/src/utils/get-extension-config.js

```diff
@@ -6,22 +6,7 @@
  */
 
 // Platform Imports
-import {getConfig} from '@salesforce/pwa-kit-runtime/utils/ssr-config'
+import {extensionConfigs} from '@salesforce/pwa-kit-extension-sdk/react'
+import extensionMeta from '../../extension-meta.json'
 
-export const getExtensionConfig = () => {
-    const extensions = getConfig()?.app?.extensions || []
-    const extensionEntry = extensions.find(
-        ([id]) => id === '@salesforce/extension-chakra-storefront'
-    )
-
-    // TODO: This isn't the right way to do this for multiple reasons.
-    // 1. The config isn't always in tupal format so this might break.
-    // 2. The config isn't representitive of the actual configuration as it might not include defualt values
-    // that aren't supplied in the configuration file.
-    //
-    // NOTE: We should instead be getting the configuration from the extension instance via the getConfig method.
-    // We can get the extensions via the getApplicationExtensions function, but that is asyn, or get it via the
-    // useApplicationExtensions hook. But that would me that we have to refactor all the multi-site work as it
-    // is a set of utilities that doesn't have access to react context to get the config, so we'd have to pass it in.
-    return extensionEntry[1]
-}
+export const getExtensionConfig = () => extensionConfigs[extensionMeta.id]
```

Comment:
Bonus: updated this `getExtensionConfig` with a synchronous, more direct way of getting the config. Take a look at the small change in the `packages/pwa-kit-extension-sdk/src/configs/utils.ts` to export this config.

Reply:
Thanks.. that was on my list of things to look into doing